layouts/partials/article_metadata.html

```diff
@@ -4,13 +4,25 @@
 
   {{ if $.Params.authors }}
   {{ $authorLen := len $.Params.authors }}
-  <div itemprop="author">
+  {{ if eq $authorLen 0}}
+  <div itemscope itemprop="author" itemtype="http://schema.org/Person">
+    <span itemprop="name">{{ $.Site.Params.name }}</span>
+  </div>
+  {{ else }}
+  <div>
     {{ range $k, $v := $.Params.authors }}
-      {{- $v -}}
+      <span itemscope itemprop="author" itemtype="http://schema.org/Person">
+        <span itemprop="name">{{- $v -}}</span>
+      </span>
       {{- if lt $k (sub $authorLen 1) -}}, {{ end }}
     {{ end }}
   </div>
   {{ end }}
+  {{ else }}
+  <div itemscope itemprop="author" itemtype="http://schema.org/Person">
```

Comment:
Please make this hidden like it was further down the page before - i.e. use span and meta for the HTML tags.